### PR TITLE
fix: dotenv-expand only works for the .env file

### PR DESCRIPTION
**Pull request by Jolg42**
See tracking issue https://github.com/prisma/docs/issues/2835

### CODE REVIEW

File: content/200-concepts/100-components/01-prisma-schema/index.mdx [hunk 1]
```diff
@@ -195,19 +195,19 @@ datasource db {
 }
 ```
 
-In version 2.7.0 and later, you can expand variables using the format specified by [dotenv-expand](https://github.com/motdotla/dotenv-expand). This is useful when dealing with PaaS or similar products. such as Heroku):
+The `env()` function can be used in the following places:
+
+- A datasource url
+- Generator binary targets
+
+In version 2.7.0 and later, variables stored in the `.env` file can be expanded using the format specified by [dotenv-expand](https://github.com/motdotla/dotenv-expand). This is useful when dealing with PaaS or similar products. such as Heroku):
```
Comment (andrew-walford-prisma) [inline content/200-concepts/100-components/01-prisma-schema/index.mdx]:
```suggestion
In version 2.7.0 and later, you can expand any variables stored in the `.env` file. Use the format specified by [dotenv-expand](https://github.com/motdotla/dotenv-expand). This is useful when dealing with PaaS or similar products such as Heroku):
```

File: content/200-concepts/100-components/01-prisma-schema/index.mdx [hunk 2]
```diff
@@ -195,19 +195,19 @@ datasource db {
 }
 ```
 
-In version 2.7.0 and later, you can expand variables using the format specified by [dotenv-expand](https://github.com/motdotla/dotenv-expand). This is useful when dealing with PaaS or similar products. such as Heroku):
+The `env()` function can be used in the following places:
```
Comment (andrew-walford-prisma) [inline content/200-concepts/100-components/01-prisma-schema/index.mdx]:
```suggestion
You can use the `env()` function in the following places:
```

**Review by andrew-walford-prisma — COMMENTED** [review]
I made a few suggestions to comply with the docs style guide. It otherwise LGTM!

File: content/200-concepts/100-components/01-prisma-schema/index.mdx [hunk 3]
```diff
@@ -195,19 +195,19 @@ datasource db {
 }
 ```
 
-In version 2.7.0 and later, you can expand variables using the format specified by [dotenv-expand](https://github.com/motdotla/dotenv-expand). This is useful when dealing with PaaS or similar products. such as Heroku):
+You can use the `env()` function in the following places:
+
+- A datasource url
+- Generator binary targets
+
+In version 2.7.0 and later, you can expand any variables stored in the `.env` file. Use the format specified by [dotenv-expand](https://github.com/motdotla/dotenv-expand). This is useful when dealing with PaaS or similar products such as Heroku):
```
Comment (janpio) [inline content/200-concepts/100-components/01-prisma-schema/index.mdx]: Opening `(` is missing (as it was before as well).

Comment (janpio) [inline content/200-concepts/100-components/01-prisma-schema/index.mdx]: Honestly I think this information is just wrong here. This has nothing to do with accessing env vars from the Prisma schema. This is functionality we have only for the `.env` file and also only works there. It should be in some docuemntation about our `.env` usage and not here.

Comment (Jolg42) [reply content/200-concepts/100-components/01-prisma-schema/index.mdx]:
```suggestion
In version 2.7.0 and later, you can expand any variables stored in the `.env` file. Use the format specified by [dotenv-expand](https://github.com/motdotla/dotenv-expand). (This is useful when dealing with PaaS or similar products such as Heroku):
```

Comment (Jolg42) [reply content/200-concepts/100-components/01-prisma-schema/index.mdx]: I removed it and made the more info link clearer

**Review by janpio — APPROVED** [review]
👍 

Is the expand information present at the location this links to?

File: content/200-concepts/100-components/01-prisma-schema/index.mdx (hunk 3 above)
Comment (Jolg42) [reply content/200-concepts/100-components/01-prisma-schema/index.mdx]: Makes sense to clean it up like this

**Comment by Jolg42** [comment]
@janpio Yes! https://www.prisma.io/docs/guides/development-environment/environment-variables#expanding-variables

**Comment by janpio** [comment]
We should never have had this confusing bit here then 🤷 

Thanks for finally digging into this and fixing it!

File: content/200-concepts/100-components/01-prisma-schema/index.mdx (hunk 3 above)
Comment (andrew-walford-prisma) [reply content/200-concepts/100-components/01-prisma-schema/index.mdx]: I suggest that we lose the brackets around the third sentence here.

File: content/200-concepts/100-components/01-prisma-schema/index.mdx [hunk 4]
```diff
@@ -195,20 +195,12 @@ datasource db {
...
-```
-
-The `env()` function can be used in the following places:
+You can use the `env()` function in the following places:
 
 - A datasource url
 - Generator binary targets
 
-See [Environment variables](/guides/development-environment/environment-variables) from more information.
+See [Environment variables](/guides/development-environment/environment-variables) for more information about how to use a `.env` file during development.
```
Comment (Jolg42) [inline content/200-concepts/100-components/01-prisma-schema/index.mdx]:
```suggestion
See [Environment variables](/guides/development-environment/environment-variables) for more information about how to use an `.env` file during development.
```

**Pull request merged**